src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -0,0 +1,285 @@
+{
+    "$schema": "./aws-deploy-recipe-schema.json",
+    "Id": "AspNetAppAppRunner",
+    "Version": "0.1.0",
+    "Name": "ASP.NET Core App to AWS App Runner",
+    "DeploymentType": "CdkProject",
+    "DeploymentBundle": "Container",
+    "CdkProjectTemplate": "../CdkTemplates/AspNetAppAppRunner",
+    "CdkProjectTemplateId": "netdeploy.AspNetAppAppRunner",
+    "Description": "Recommended for ASP.NET Core applications to be deployed in a fully management environment as a container image. If your project does not contain a Dockerfile, one will be generated for the project.",
```

Comment:
nit: for consistency with other descriptions, can we not start the sentence with "Recommended" ?

Reply:
We need to also add a ShortDescription.

Reply:
Added short description and cleaned up the description.